### DISCUSSION TITLE
Not working ?

### DISCUSSION THREAD

**Discussion by JesusPanda** [dsdanielpark/Bard-API#200] [Q&A]
Hi, i tried to use the bard api, i followed the "tutorial" step by step but when i try to exetute the program it tells me that there are 3 errors: 
line 151, in _get_snim0e
    raise Exception(
Exception: SNlM0e value not found. Double-check __Secure-1PSID value or pass it as token='xxxxx'.
# i have tried to fix it but my key look, correct and it ends in a .
line 45, in __init__
    self.SNlM0e = self._get_snim0e()
                  ^^^^^^^^^^^^^^^^^^
# i have no idea what this error and i don't feel confident in modifing the code 
line 5, in <module>
    Bard().get_answer("나와 내 동년배들이 좋아하는 뉴진스에 대해서 알려줘")['content']
    ^^^^^^
#this one is hard to understand and at first i tried to re-download the file but it did't fix it 
# hope you can help me, tx

**Comment by dsdanielpark** [comment]
During the optimization and refactoring process, I identified formatting and import errors. These issues have been addressed through QA to prevent recurrence.

Please use 0.1.38 or 0.1.33 version.

**Reply by JesusPanda** [reply]
tx
